test/OpenTelemetry.Tests.Stress.Metrics/Program.cs

```diff
@@ -25,15 +25,15 @@ public partial class Program
     private const int ArraySize = 10;
 
     // Note: Uncomment the below line if you want to run Histogram stress test
-    // private const int MaxHistogramMeasurement = 1000;
+    private const int MaxHistogramMeasurement = 1000;
```

Comment:
i figured its easy to target Exemplar + Histogram for the stress test now as that is the area being actively worked on now.